test/helpers.ts

```diff
@@ -8,25 +9,36 @@ export type Response = {
 export type Request = {
   method: string;
   path: string;
-  params: Record<string, unknown>;
-  data?: Record<string, unknown>;
+  params?: Record<string, unknown>;
+  data?: unknown;
+};
+
+export const MOCK_FETCH_CONFIG = {
+  baseURL: "https://api.com",
```

Comment:
I slightly prefer to use one of the test domains for this if we can:
```suggestion
  baseURL: "https://example.net",
```